.github/workflows/image-optimization.yml

```
@@ -23,0 +1,26 @@
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    name: calibreapp/image-actions
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v4
+
+      - name: Compress Images
+        uses: calibreapp/image-actions@main
+        with:
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          ignorePaths: src/test/**
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)